lib/noun-project-api/connection.rb

```diff
@@ -0,0 +1,14 @@
+module NounProjectApi
+  # Basic connection methods and setup.
+  module Connection
+    attr_accessor :token, :secret, :access_token
+
+    def initialize(token, secret)
+      @token = token
+      @secret = secret
+      fail(ArgumentError, 'Missing token or secret') unless @token && @secret
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.